ui/tests/helpers/components/mount-backend-form-selectors.ts

```diff
@@ -0,0 +1,9 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+export const MOUNT_BACKEND_FORM = {
+  header: '[data-test-mount-form-header]',
```

Comment:
since there's only one header per page, could we update the selector to use the general selector `data-test-page-title`